packages/cli/src/commands/source/push/add.ts

```diff
@@ -88,7 +89,11 @@ export default class SourcePushAdd extends Command {
       );
     }
     const cfg = await new AuthenticatedClient().cfg.get();
-    const source = new Source(cfg.accessToken!, cfg.organization);
+    const {environment, region} = this.configuration.get();
```

Comment:
🤔  Isn't this information already available on the `cfg` retrieved from `AuthenticatedClient` ?

So this + line 181-183 could be removed.

Reply:
yes you're right. my bad